### ISSUE TITLE
import error 

### ISSUE THREAD

**Issue by mo-nonym** [tatarize/vpype-dxf#3]
Error: 
```
from ezdxf.tools.rgb import DXF_DEFAULT_COLORS, int2rgb
ModuleNotFoundError: No module named 'ezdxf.tools.rgb'
```
Solution:
dread.py line 5
```
from ezdxf.colors import DXF_DEFAULT_COLORS, int2rgb
```
P.S. I should learn how github works...

**Comment by tatarize** [comment]
Yeah, I'll fix this shortly. There was a breaking change in ezdxf 0.16.5+ when those imports were moved. Other code that used that stuff needed some changes. I'll have it fixed in a couple hours.

**Comment by tatarize** [comment]
Fixed in 0.0.4
